HHS/Head-Start-TTADP
# [TTAHUB-448] Specialist name filter

## Description of change

Add a 'Specialist name' filter to the AR landing and Regional Dashboard. We only add this filter if the user has the ability to approve reports.

## How to test

Create a user with the ability to approve reports. On both the AR Landing and Regional Dashboard create a filter using the new 'Specialist name' filter. Ensure relevant results are returned. 

Create a user who does NOT have the ability to approve reports. Both the AR Landing and Regional Dashboard should NOT have the ability to add the 'Specialist name' filter.

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-448


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
